charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```diff
@@ -223,14 +238,17 @@ spec:
             - containerPort: {{ .Values.alertManager.port }}
               name: alerts
               protocol: TCP
+            - containerPort: {{ .Values.healthProbes.port | default 8081 }}
```

Comment:
The `| default 8081` part of this line is duplicating the specified default value here https://github.com/banzaicloud/koperator/blob/1288fcf55626eaa4c77cd556bcc03a5c2de42d97/charts/kafka-operator/values.yaml#L77

This is unnecessary and creates potential confusion, IMO we should only use either way to set the default value

See`webhook.serverPort` as an example:
https://github.com/banzaicloud/koperator/blob/ca1483ec553d99dc6b3df8d6f76f06d24fca0f7e/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L231

https://github.com/banzaicloud/koperator/blob/ca1483ec553d99dc6b3df8d6f76f06d24fca0f7e/charts/kafka-operator/values.yaml#L42